src/controllers/recipe.controller.ts

```diff
@@ -20,6 +21,8 @@ export class RecipeController {
     try {
       const { title, description, preparing, ingredients, url } =
         req.body as CreateRecipeBody;
+      const { _id } = req.user as UserDocument;
```

Comment:
Here I'd either change `Request` to `AuthorizedRequest` or some similar interface with defined `user` property or modify globals as we previously did (the structure is a little bit different).